lib/Test/Nginx/Util.pm

```diff
@@ -1205,6 +1205,11 @@ sub run_test ($) {
         }
     }
 
+    if (defined $block->stop_after_request && $RepeatEach != 1){
+       bail_out("\"stop_after_request\" flag is enable, nginx will be stoped after request. \"repeat_each\" number must be set to 1.");
```

Comment:
Grammar error:

"is enable" => "is enabled"
"be stoped" => "be stopped"
"after request" => "after the first request"